### PR TITLE
Fetch topic from GraphQL

### PR DESCRIPTION
**Pull request by aaronschachter**
#### What's this PR do?

Opened in tandem with https://github.com/DoSomething/graphql/pull/40, this PR fetches a topic via `topic(id)` GraphQL request instead of a [`GET /topics/:id` Gambit Content request](https://github.com/DoSomething/gambit-content/blob/master/documentation/endpoints/topics.md) and caches the result. It also resolves some small breaking changes:

- The `templates` object property no longer exists, instead the template values are defined as properties on each Topic type

-  The `askYesNo` type accounts for this by adding new `saidYesTopic` and `saidNoTopic` object properties

- The campaign `status` field no longer exists, instead the campaign helper checks the `endDate` property, porting over [Gambit Content helper](https://github.com/DoSomething/gambit-content/blob/6772d603837ab419b3376d3068c43c7b01d68cad/lib/helpers/campaign.js#L127)

TODO after merge:

- Rename topic types from Contentful type names to GraphQL type names (e.g. rename all instances of `askYesNo` type value to  `AskYesNoBroadcastTopic`)

- Support for `askVotingPlanStatus` broadcast topic: We won't be using this any time in the near future

#### How should this be reviewed?

Verify expected replies for inbound messages in each topic:

- Auto reply topic without a campaign: say No to broadcast 2Tlt66Th0VYCqX8QB4VjT4

- Auto reply topic with a campaign - text TESTING to join topic 52hS9t2Uak0kaiCME0ImCi, which creates a signup for campaign 8303 then sends an autoReply for all other inbound messages while in topic

- Photo post topic - text MIRROR and verify expected behavior

- Text post topic - text CHEESE and verify expected behavior

- Invalid ask yes no response - Say "France" to broadcast 2Tlt66Th0VYCqX8QB4VjT4, verify `invalidAskYesNoResponse` template is sent

- When current topic campaign end date has passed, the `closedCampign` reply is sent. (Note: there's a bug when you first send the campaign keyword, the `internalTitle` is not available in Gambit Content API. The subsequent message you send when in the closed campaign is served from GraphQL:
```
You> power

Bot> Sorry,  is no longer available. Text Q if you have a question.

You> i should see the title now

Bot> Sorry, You've Got the Power run 1 is no longer available. Text Q if you have a question.
```

#### Any background context you want to provide?

This a first (pretty big!) step towards using GraphQL to deprecate the Gambit Content API

#### Checklist
- [x] Documentation added for new features/changed endpoints.
- [x] Tests added/updated for new features/bug fixes.
- [x] ENV variables added/updated for new features/bug fixes.
- [x] Tested on staging.
- [x] Pinged a PM if this is a larger PR that would benefit from some additional testing love.

### CODE REVIEW

**Comment by aaronschachter** [comment]
Still work in progress -- Getting 503's in GraphQL when saying yes to broadcast `2oPo0wYLzymCphGZXJI8b0` 🤕

**Comment by aaronschachter** [comment]
Timeouts for broadcast replies to 2oPo0wYLzymCphGZXJI8b0 are resolved per https://github.com/DoSomething/graphql/pull/40/commits/cf640e31d9c09afcede9de30cc5aaf06fe57e9c3. Replying to this broadcast will return a 500 error, as the saidYesTopic's campaign (id 9002) exists in Rogue production but not Rogue QA.

File: test/unit/lib/lib-helpers/campaign.test.js [hunk 1]
```diff
@@ -39,14 +38,10 @@ test('fetchById calls gambitContent.fetchCampaignById', async () => {
 });
 
 // isClosedCampaign
-test('isClosedCampaign should return true when campaign is active', (t) => {
-  const result = campaignHelper.isClosedCampaign(campaignFactory.getValidCampaign());
-  t.falsy(result);
+test('isClosedCampaign should return false when campaign.endDate undefined', (t) => {
+  t.falsy(campaignHelper.isClosedCampaign(campaignFactory.getValidCampaign()));
```
[the remark this answers is not in the document]
Comment (aaronschachter) [reply test/unit/lib/lib-helpers/campaign.test.js]: I agree! I made this update.

File: config/lib/helpers/replies.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ module.exports = {
   },
   campaignClosed: {
     name: 'campaignClosed',
-    text: process.env.GAMBIT_CONVERSATIONS_CAMPAIGN_CLOSED_TEXT || `Sorry, {{topic.campaign.title}} is no longer available. ${supportText}`,
+    text: process.env.GAMBIT_CONVERSATIONS_CAMPAIGN_CLOSED_TEXT || `Sorry, {{topic.campaign.internalTitle}} is no longer available. ${supportText}`,
```
Comment (DFurnes) [inline config/lib/helpers/replies.js]: Internal title isn't intended to be user-facing (e.g. `Grab the Mic 2019-01` or `Friends On Lock run 1`). Can this just say "Sorry, this campaign is no longer available" instead?

**Pull request merged**